include/tlog/errs.h

```diff
@@ -141,4 +141,68 @@ extern void tlog_errs_pushf(struct tlog_errs **perrs, const char *fmt, ...)
  */
 extern int tlog_errs_print(FILE *stream, const struct tlog_errs *errs);
 
+/**
+ * Push an error to a stack, with message as a constant string
+ * and then go to cleanup
+ *
+ * @param fmt   The message string to copy and put into the pushed error.
+ *              Cannot be NULL.
+ */
+#define TLOG_ERRS_RAISES(_fmt) \
+     do {                                       \
+         tlog_errs_pushs(perrs, _fmt);          \
+         goto cleanup;                          \
+     } while (0)
+
+/**
+ * Push an error to a stack, with message being a global return code
+ * description, update the local grc variable
+ * Push an error to a stack, with message as a constant string
+ * and then go to cleanup
```

Comment:
Please don't just write separate sentences about what this function is doing. Instead start it with something like: "Push two errors on the stack: ..."

Reply:
Also mention assigning `_grc` to `grc`.